apps/filebrowser/src/filebrowser/conf.py

```diff
@@ -40,6 +40,18 @@
   default=None,
   type=str)
 
+FILE_UPLOAD_CHUNK_SIZE = Config(
+  key="file_upload_chunk_size",
+  default=5000000,
+  type=int,
+  help=_('flag to change the chunk size of file upload'))
+
+CONCURRENT_MAX_CONNECTIONS = Config(
```

Comment:
Suggestion: If this is for upload, then we should mention something related to file upload in the config I feel.

Reply:
changed it.